### DISCUSSION TITLE
Allow for a multi user installation

### DISCUSSION THREAD

**Discussion by Carsten42** [Kong/insomnia#4413] [Ideas]
Hello Insomnia Team,
my company uses Insomnia on training environments, where virtual machines are deployed with a number of user accounts. Right now all these users need to install the software separately. We'd like to have the option to install the software for all users rather than just one of them.
I have read a bit and found out that the installer module called "squirrel" does not allow for that yet. But it would be nice and time saving to have that feature from an administrative point of view. I have been asked by support to file that request here to give it a forum.
Best
Carsten

**Comment by dimitropoulos** [comment]
it seems like https://github.com/Kong/insomnia/discussions/3658 would solve this.  what do you think?

**Reply by Carsten42** [reply]
Yes, I agree.
Can you mark this as a duplicate from #3658 and close this one?
